NethServer/Firewall.pm

```diff
@@ -1013,12 +1027,14 @@ sub countReferences($$)
     my @tcrules = $self->getTcRules(); 
     my @pfrules = $self->getPortForwards();
     my @bypass = $self->getBypassRules();
+    my @aliases = $self->getAliases();
     push(@fwrules, @tcrules);
     push(@fwrules, @pfrules);
     push(@fwrules, @bypass);
+    push(@fwrules, @aliases);
```

Comment:
This is a bit misleading.

Please separate the logic for aliases by creating a new loop after the one `@fwrules` array.